container/image.bzl

```diff
@@ -175,15 +177,19 @@ def _impl(ctx, files=None, file_map=None, empty_files=None, empty_dirs=None,
     entrypoint: str List, overrides ctx.attr.entrypoint
     cmd: str List, overrides ctx.attr.cmd
     symlinks: str Dict, overrides ctx.attr.symlinks
-    output: File to use as output for script to load docker image
     env: str Dict, overrides ctx.attr.env
     layers: label List, overrides ctx.attr.layers
     debs: File list, overrides ctx.files.debs
     tars: File list, overrides ctx.files.tars
+    output_executable: File to use as output for script to load docker image
+    output_image: File, overrides ctx.outputs.out
```

Comment:
Can we rename this `output_tarball` for clarity

Reply:
Done.